_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Bitcoin Optech Newsletter #56'
+permalink: /en/newsletters/2019/07/24/
+name: 2019-07-24-newsletter
+slug: 2019-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes progress on signet and an idea for just-in-time
+routing on LN.  Also included are our regular sections about bech32
+sending support and notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.1 release candidates:** this upcoming
+  maintenance release fixes several bugs that affected certain RPC
+  commands and caused unnecessarily high CPU use in certain cases.
+  Production users are encouraged to test the current [release
+  candidate][core rc] to ensure that it operates as expected.
+
+## News
+
+- **Progress on signet:** signet is a testnet alternative where valid
+  blocks are signed by a trusted authority that generally produces
+  a regular series of blocks but which may occasionally produce forks leading
+  to block chain reorganizations.  This avoids commonly-encountered
+  problems on testnet such as too-fast block production, too-slow block
+  production, and reorganizations involving thousands of blocks.
+  Subsequent to our earlier report on signet in [Newsletter #37], its
+  author has created an implementation documented on a [wiki
```

Comment:
Split these out: '... has created an implementation, documented it on a [wiki page][signet wiki], opened a [PR]...'